tests/core/snapd-failover/task.yaml

```
@@ -11,6 +11,10 @@ debug: |
     journalctl -u snapd.failure.service
     journalctl -u snapd.socket || true
     ls -l /snap/snapd/
+    cat /etc/systemd/system/snapd.service
+    cat /etc/systemd/system/usr-lib-snapd.mount
+    /snap/snapd/x1/usr/bin/snap debug state /var/lib/snapd/state.json || true
+    /snap/snapd/x1/usr/bin/snap debug state --change="$(snap debug state /var/lib/snapd/state.json|tail -n1|awk '{print $1}')" /var/lib/snapd/state.json || true
```

hmm shouldn't this also use `/snap/snapd/x1/usr/bin/snap` in the subshell? i.e.

```suggestion
    /snap/snapd/x1/usr/bin/snap debug state --change="$(/snap/snapd/x1/usr/bin/snap debug state /var/lib/snapd/state.json|tail -n1|awk '{print $1}')" /var/lib/snapd/state.json || true
```